### DISCUSSION TITLE
Clear All Data

### DISCUSSION THREAD

**Discussion by Scott8586** [mcguirepr89/BirdNET-Pi#873] [Q&A]
Before I go and do this, I'm writing to ask, what exactly does "Clear All Data" do - does it only erase the recordings and database, or does it also remove settings and preferences? I'd like to keep my current settings, but start recordings and identifications from scratch.

**Comment by ehpersonal38** [comment]
Hi @Scott8586 - the clear all data function works as you expect, it will erase only the recordings and database while leaving your `birdnet.conf` (your settings file) untouched.

**Comment by nkwsy** [comment]
Is there a way to have the program reanalyze previous recordings with updated model? I have been getting an outrageous amount of Tree Pipits for Chicago... and these Pipit's sound a lot like geese

**Reply by ehpersonal38** [reply]
That could be an interesting feature... but unfortunately not as of now.
